src/lib/icons.tsx

```diff
@@ -54,6 +66,23 @@ export const BackIcon = (props: JSX.HTMLAttributes) => (
   </Icon>
 );
 
+export interface FlipIconProps extends JSX.HTMLAttributes {
+  direction: 'horizontal' | 'vertical';
+}
+
+export const FlipIcon = (props: FlipIconProps) => {
+  const { direction, ...otherProps } = props;
+
+  return (
+    <Icon {...otherProps}>
+      <path
+        transform={`rotate(${ direction === 'horizontal' ? 90 : 0 })`}
```

Comment:
This transform is almost certainly wrong, but I'll catch it when I add the buttons properly.